Web/Pages/CCGMember/Compare/Delta.cshtml

```diff
@@ -0,0 +1,269 @@
+﻿@page
+@model Web.Pages.CCGMember.Compare.DeltaModel
+@inject IViewLocalizer Localizer
+@{
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+    var current = Model.CurrentPosition;
```

Comment:
All the variables (except isEnglish) aren't needed since they can be accessed on the page with Model.<X>